test/logstream/interface.go

```diff
@@ -54,3 +63,18 @@ var (
 	stream streamer
 	once   sync.Once
 )
+
+type shim struct {
+	logstreamv2.Source
+}
```

Comment:
```suggestion
type shim logstreamv2.Source
```
Will this work?

Reply:
I don't think so, because you can't add new methods to the type? I am not totally sure.

Reply:
you define a new type :)
Let me try, sinc e it merged

Reply:
OK, it would've worked if logstream2.Source wasn't an interface, but a concrete type.